frontend/src/metabase/dashboard/actions/writeback.ts

```diff
@@ -279,5 +285,6 @@ export const executeRowAction = ({
         }),
       );
     }
-  };
+    return { success: false, error: message, message };
```

Comment:
returning the error message like this allows us to show errors in the modal forms as well as inline forms.

![Screen Shot 2022-09-21 at 4 50 27 PM](https://user-images.githubusercontent.com/30528226/191623938-528252ad-20c7-404c-8f6e-c7c9bf631177.png)